spec/models/raw_input_changes_spec.rb

```diff
@@ -277,10 +277,8 @@
           organization = Fabricate :organization
           Fabricate :website, organization: organization, content: website
 
-          source = Fabricate :source
           import = Fabricate(
             :import,
-            source: source,
```

Comment:
I saw that it could be removed because the `:import` fabricator already creates it by default. So, it's redundant, unless you need to test on `source` itself.